app/assets/stylesheets/mixins.scss

```
@@ -20,6 +20,14 @@
   }
 }
 
+%consul-gradient {
+  background: #36424a;
+  background: -moz-linear-gradient(-45deg, #36424a 0%, #3a778b 100%);
+  background: -webkit-linear-gradient(-45deg, #36424a 0%,#3a778b 100%);
+  background: linear-gradient(135deg, #36424a 0%,#3a778b 100%);
```

```suggestion
  background: -moz-linear-gradient(-45deg, #36424a 0%, #3a778b 100%);
  background: -webkit-linear-gradient(-45deg, #36424a 0%,#3a778b 100%);
  background: linear-gradient(135deg, #36424a 0%,#3a778b 100%);
  background-color: #36424a;
```

This way we don't get the warnings you mentioned about the contrast.